samples/rag-storage-document-loaders/embed.py

```diff
@@ -48,8 +51,22 @@ def main() -> None:
         embedding_function=embed_model,
     )
 
-    for batch in batched(loader.lazy_load(), _EMBED_BATCH_SIZE):
-        azure_search.add_documents(list(batch))
+    text_splitter = RecursiveCharacterTextSplitter(
```

Comment:
Let's move the recursive text splitter declaration to in between the loader and the embeddings model to better represent where it is used in this pipeline.